### ISSUE TITLE
Jacket deleting all my indexers when I update

### ISSUE THREAD

**Issue by linkandzelda91** [Jackett/Jackett#11253]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug.

### Environment

**OS**: ubuntu, lxd container

**.Net Runtime**: mono

**.Net Version**: mono

**Jackett Version**: well, I restore to a snapshot from august to try and fix this, but the latest version
 Jackett Version v0.17.638 

**Last Working Jackett Version**:
probably the last one, but my snapshot is all the way from august so it loads as:
 0.16.1023.0

this one shows all my indexers
**Are you using a proxy or VPN?** [yes/no]
no
### Description

[List steps to reproduce the error and details on what happens and what you expected to happen]
go to jacket GUI. all my indexers are gone. I restore my snapshot, they are all there but I'm on an older version. when i click update and then reload, they are all gone again.

### Logged Error Messages

https://pastebin.com/b1z1kSMx
logs from GUI

### Screenshots

[Place any screenshots of the issue here if needed]

**Comment by ilike2burnthing** [comment]
I'm not familiar with lxd containers, but assuming it's similar enough to Docker, check your mapped folders, as your logs are showing a completely fresh install, hence all settings being default and you having no indexers.

**Comment by linkandzelda91** [comment]
> 
> 
> I'm not familiar with lxd containers, but assuming it's similar enough to Docker, check your mapped folders, as your logs are showing a completely fresh install, hence all settings being default and you having no indexers.

thank you for the reply. there are no mapped drives, either before when it is working properly after a restore to the older version, or after when I rupdate and the indexers are gone. is there a command to make jacket show where it's current config is located? I can check the old and new and see if they are different maybe?

**Comment by linkandzelda91** [comment]
oooooh hang on I'm onto something standby.

**Comment by linkandzelda91** [comment]
https://imgur.com/a/VGEMbb4

pics of my adventure

seems that before the update, my Jacket was looking in another dir for the config files. and after the update it changed that dir on its own?

anyway I moved over the config files and now I have my indexers back. still wonder why it moved them tho (they were, admittedly in a weird place to start. when I did the install last year I thought I followed the official guide
